### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards> Expenses Report]: Name is not defined for the 'Expand and Collapsed' button present under Amount column. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8973]
### Target Platforms

UWP

### SDK Version

1.0.1.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Universal
OS Build: 24H2 (OS Build 26252.5000)
App Version: 1.0.1.0
Screen Reader: Narrator

**Repro Steps:**
1. Open the Adaptive Cards Universal application.
2. Adaptive Card UWP Visualizer screen starts appearing.
3. Navigate to Expenses Report.Json button by using Tab key and activate it.
4. Expenses Report adaptive card will appear on right sight of the pane.
5. Navigate to Expenses Report card by using Tab key.
6. Press the tab key and navigate to the 'Expand' button and observe the issue.

**Actual Result:**
While navigating the expenses report card, screen reader is not conveying the name of the 'Expand or collapse' button as no name is defined for the same. When the focus lands on it then it is announcing only as 'Button.'

Also, after activating the 'Expand or collapse' button screen reader is not conveying the changed name information. It is remain silent.

**Expected Result:**
Name should be defined for the Expand or collapse button so that when the screen reader focus lands on it then it should announce the information as **'Expand Air Travel expenses dollar 300 button.'**

Also, when user activates Expand button then it should announce as 'Expanded' and vice versa.

**Note:**
Same issue is repro with NVDA.

**User Impact:**
This will impact the screen reader user as they will not be able to know the name of the control due to this they will not able to get to know by activating the control what action will be performed.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

![Name is not defined for the expand collapsed button](https://github.com/user-attachments/assets/dbcc7a58-da34-45ef-86e9-9c0052d59855)

https://github.com/user-attachments/assets/0675c177-f835-44ae-8e92-1bfe0c5727c1

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is still repro on the updated build.

**Build Version:**
1.0.2.0

**Snippet:**
![Image](https://github.com/user-attachments/assets/6be54c07-ca16-451d-aa5b-8444e4eac287)

**Comment by vagpt** [comment]
Closing this issue here as of now this issue is still repro'ing on the latest build (1.0.2.0) and now this issue can be tracking with below URL.

https://github.com/microsoft/AdaptiveCards/issues/9076
